sycl/plugins/unified_runtime/ur/adapters/level_zero/ur_level_zero_common.cpp

```diff
@@ -45,3 +45,214 @@ ur_result_t ze2urResult(ze_result_t ZeResult) {
   }
   return It->second;
 }
+
+void urPrint(const char *Format, ...) {
+  if (UrL0Debug & UR_L0_DEBUG_BASIC) {
+    va_list Args;
+    va_start(Args, Format);
+    vfprintf(stderr, Format, Args);
+    va_end(Args);
+  }
+}
+
+void zePrint(const char *Format, ...) {
+  if (ZeDebug & ZE_DEBUG_BASIC) {
+    va_list Args;
+    va_start(Args, Format);
+    vfprintf(stderr, Format, Args);
+    va_end(Args);
+  }
+}
```

Comment:
why do you need 2 of them?

Reply:
@smaslov-intel this is for now. zePrint will go away in a few patches after.

Reply:
OK, I am fine with temp duplication